packages/tools/src/utilities/planar/filterAnnotationsWithinPlan.ts

```diff
@@ -0,0 +1,77 @@
+import { vec3 } from 'gl-matrix';
+import { CONSTANTS, metaData } from '@cornerstonejs/core';
+import type { Types } from '@cornerstonejs/core';
+import { Annotations, Annotation } from '../../types';
+
+const { EPSILON } = CONSTANTS;
+
+const PARALLEL_THRESHOLD = 1 - EPSILON;
+
+/**
+ * given some `Annotations`, and the slice defined by the camera's normal
+ * direction and the spacing in the normal, filter the `Annotations` which
+ * is within the slice.
+ *
+ * @param annotations - Annotations
+ * @param camera - The camera
+ * @param spacingInNormalDirection - The spacing in the normal direction
+ * @returns The filtered `Annotations`.
+ */
+export default function filterAnnotationsWithinSamePlan(
```

Comment:
I'm not sure why we need this function, since annotationWithinSameSlice already checks for the same plane (`annotationsWithParallelNormals`)

Reply:
`annotationWithinSameSlice` was returning the annotations that were on the same plane and slice aswell, but we wanted all the annotations for the same plane only.